alacritty_macos.yml

```diff
@@ -200,5 +200,10 @@ mouse_bindings:
 
 # Shell
 #
-# You can set this to a path to your favorite shell, e.g. /bin/fish
-shell:
```

Comment:
Oh, missed this question and saw the PR. Simply having it commented out is great.